lib/foreman_maintain/executable.rb

```diff
@@ -40,5 +43,17 @@ def __run__(execution)
       @_execution = execution
       run
     end
+
+    # method defined both on object and class to ensure we work always with object
+    # even when the definitions provide us only class
+    def ensure_instance
+      self
+    end
+
+    class << self
+      def ensure_instance
```

Comment:
The this allows to use both class and instance of class in `next_steps` definition.

With this, you can do both:

```
:next_steps => [Producedures::ForemanTasksResume]
# as well as
:next_steps => [Producedures::ForemanTasksResume.new]
```

What do you think? I guess we could neforce only using instances in this cases, to not make to much confusion over how it actually works